.dlint.json

```diff
@@ -4,6 +4,8 @@
     "include": [
       "ban-untagged-todo"
     ],
-    "exclude": []
+    "exclude": [
+      "no-invalid-triple-slash-reference"
+    ]
```

Comment:
If I understand correctly, `no-invalid-triple-slash-reference` rule is for Deno users, not for the internal code of Deno, so I added it here. Please correct me if I'm wrong.

Reply:
That's correct